### DISCUSSION TITLE
How to set Effect through code?

### DISCUSSION THREAD

**Discussion by XDgl233** [AvaloniaUI/Avalonia#12658] [Q&A] [answered]
Hello! I am currently working on some custom properties for my templated control. And I chose to expose the children's effect property via custom properties on the template. But I can't find a way to parse a string like I do in xaml editor for the effect property. Very appreciate any help possible :)

**Comment by timunie** [comment, marked as answer]
In code you need to set create the actual Effect, not a string.

**Reply by XDgl233** [reply]
Thank you, I looked into the code, and used DropShadowEffect class.
